src/index.html

```diff
@@ -4,24 +4,403 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, initial-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    />
+    <title>Moyo header</title>
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
```

Comment:
The href attribute in the link tag should point to a CSS file, not a SCSS file. Please ensure that the correct CSS file is linked.